lib/Bindings/Python/dialects/om.py

```diff
@@ -122,6 +134,22 @@ def __iter__(self):
       yield (wrap_mlir_object(i), self.__getitem__(i))
 
 
+class FrozenBasePath(BaseFrozenBasePath):
+
+  def __init__(self, obj: BaseFrozenBasePath) -> None:
```

Comment:
Is it necessary to override this constructor if all it does is call `super`?

Reply:
Thanks for the suggestion